src/BabylonBindings/SceneObjects/Lightbinding.ts

```diff
@@ -0,0 +1,58 @@
+import { ShadowGenerator,  IShadowLight, PointLight, SpotLight, DirectionalLight,
```

Comment:
This file name should be `camelCase`. Currently it obeys no naming scheme at all.